scenes/game/local_game.gd

```diff
@@ -4358,12 +4425,14 @@ func continue_resolve_strike(events):
 					_append_log("Range Check: %s's %s (%s) vs %s (%s)." % [player1.name, player1.get_buddy_name(), player1.get_buddy_location(), player2.name, player2.arena_location])
 				else:
 					_append_log("Range Check: %s (%s) vs %s (%s)." % [player1.name, player1.arena_location, player2.name, player2.arena_location])
-				if in_range(player1, player2, card1):
+				if in_range(player1, player2, card1) and not card1.definition['id'] in player1.will_not_hit:
 					_append_log("%s %s hits." % [player1.name, card_db.get_card_name(card1.id)])
 					active_strike.player1_hit = true
 					active_strike.strike_state = StrikeState.StrikeState_Card1_Hit
 					active_strike.remaining_effect_list = get_all_effects_for_timing("hit", player1, card1)
 				else:
+					if card1.definition['id'] in player1.will_not_hit:
+						_append_log("Named card %s does not hit this strike!" % card_db.get_card_name(card1.id))
```

Comment:
Maybe use an else here and don't print the misses line if you print this one?
Actually, maybe combine them so it says %s %s misses (this was the named card)?
i would do this with like:
var extra_details = ""
if ....
  extra_details = " (this was the named card)"
 append_log("%s %s misses%s." % [player1.name, card_db.get_card_name(card1.id), extra_details])